api/tests/unit/domain/models/CertificationCandidate_test.js

```diff
@@ -375,6 +375,294 @@ describe('Unit | Domain | Models | Certification Candidate', function () {
         expect(call).to.not.throw();
       });
 
+      describe('validateForMassSessionImport', function () {
+        const buildCertificationCandidate = (attributes) => new CertificationCandidate(attributes);
+
+        const validAttributes = {
+          firstName: 'Oren',
+          lastName: 'Ishii',
+          sex: 'F',
+          birthPostalCode: '75001',
+          birthINSEECode: '',
+          birthCountry: 'France',
+          birthdate: '2010-01-01',
+          sessionId: 123,
+          resultRecipientEmail: 'orga@example.net',
+          billingMode: 'FREE',
+        };
+
+        context('when all required fields are presents', function () {
+          it('should return nothing when object is valid', function () {
```

Comment:
Puisque le context informe déjà que l'objet est valide on peut juste mettre
`should return undefined`